utils/staking-miner/src/main.rs

```diff
@@ -318,6 +357,10 @@ struct MonitorConfig {
 	/// The solver algorithm to use.
 	#[clap(subcommand)]
 	solver: Solvers,
+
+	/// Submission strategy to use.
```

Comment:
```suggestion
	/// Submission strategy to use. Ex: `--submission-strategy "percent-better 1"`.
```
Not sure what the convention is for clap CLIs, but it would be nice if the user could get an example arg for percent-better since its a bit tricky

Reply:
yeah, this is a bit weird maybe using an integer will work better with the CLI help menu.